packages/@ourworldindata/grapher/src/controls/Controls.scss

```diff
@@ -1,336 +1,102 @@
+        align-items: center;
+        border: 1px solid $light-stroke;
+        border-radius: 4px;
+        padding: 7px;
+        color: $dark-text;
         white-space: nowrap;
-        overflow: hidden;
-        text-overflow: ellipsis;
 
         &:hover {
-            text-decoration: underline;
+            background: $hover-fill;
+            cursor: pointer;
         }
-    }
 
-    a,
-    svg {
-        color: $controls-color;
-    }
-    svg {
-        margin-left: 0.4rem;
-        font-size: 0.8em;
-        vertical-align: unset;
-    }
-}
-
-$footerRowHeight: 32px; // keep in sync with Controls.tsx's footerRowHeight
-
-.ControlsFooter {
-    width: 100%;
-    position: absolute;
-    bottom: 0;
-    z-index: $zindex-ControlsFooter;
-    color: #777;
-    display: flex;
-    flex-direction: column;
-    justify-content: end; // align rows in the footer to the bottom
-
-    .footerRowSingle,
-    .footerRowMulti {
-        box-sizing: content-box;
-        width: 100%;
-        height: $footerRowHeight;
-    }
-
-    .footerRowSingle > * {
-        width: 100%;
-    }
-
-    .footerRowMulti {
-        display: flex;
-
-        .inline-controls {
-            flex-grow: 1;
+        &.active,
+        &:active {
+            color: $active-text;
+            background: $active-fill;
+            border: 1px solid $active-fill;
         }
 
-        nav.tabs {
-            min-width: 50%;
+        svg {
+            height: 12px;
+            margin-right: 5px;
+            vertical-align: bottom;
         }
-    }
-
-    .extraControls {
-        display: flex;
-        height: 100%;
-        align-items: center;
-        font-size: 0.8em;
-    }
 
-    .footerRowMulti .extraControls {
-        padding-left: 0.5em;
-    }
-
-    .footerRowSingle .extraControls {
-        justify-content: center;
-    }
-
-    .extraControls label {
-        margin: 0;
-        padding-left: 0.5em;
-        padding-right: 0.5em;
-    }
-
-    .extraControls button {
-        margin: 0;
-        font-size: 1em;
-        color: $controls-color;
-    }
-
-    nav.tabs ul {
-        margin: 0;
-        padding: 0;
-        display: flex;
-        width: 100%;
-        text-align: center;
-    }
-
-    nav.tabs li {
-        flex-grow: 1;
-        list-style-type: none;
-        border-left: 1px solid #e1e1e1;
-        height: $footerRowHeight;
-
-        &.download-tab-button {
-            svg {
-                margin-right: 0.15em;
-            }
+        label:hover {
+            cursor: pointer;
         }
+    }
+}
 
-        &.open-in-another-tab-button {
-            svg {
-                margin-left: 0.3em;
-            }
+//
+// Make buttons narrower by omitting their text labels as the viewport gets narrower
+// TODO: move the logic to js to calculate actual widths for different entity name lengths
+//       and the variable number of buttons present in the content switcher block
+//
+@container grapher (max-width:550px) {
```

Comment:
I think this is the first time that we use container queries in our codebase. Can you add a comment here on how this behaves if container (size?) queries are not supported?

Reply:
It's also used to step down the font size in the tooltips on small screens (where it's more just a nice-to-have feature). Here it's pretty essential though since it keeps both buttons visible, so there ought to be a fallback strategy…

Reply:
I added one more breakpoint & width-class to grapher that will kick in when container queries aren't supported